blocks/header/header.js

```diff
@@ -151,6 +172,9 @@ export default async function decorate(block) {
     });
   }
 
+  const navLinks = navSections.querySelectorAll(':scope .default-content-wrapper > ul > li > a');
+  setActiveLink(navLinks);
```

Comment:
We have an issue with this function, because in this example: https://7-menu--edge-delivery-ubs-faweb-demo--netcentric.aem.live/
Meet the team is selected and it's not corect.

Reply:
Good catch, again. I completely forgot about checking for external links.